pkg/operator/operator.go

```diff
@@ -306,7 +307,9 @@ func (c *vSphereProblemDetectorController) runChecks(ctx context.Context, cluste
 	klog.V(4).Infof("All checks complete")
 
 	results, checkError := resultCollector.Collect()
-	c.reportResults(results)
+	if !silenced {
+		c.reportResults(results)
+	}
 	var nextDelay time.Duration
```

Comment:
Should we remove the `silenced` parameter from all these functions, if `silence` state can be read from `log.Silenced` variable?

Reply:
It makes sense, I [added it ](https://github.com/openshift/vsphere-problem-detector/pull/169/commits/2392b0bea346f8faf9ca5188c7744f79de29d922) to this PR.